src/platform/common/stm32/system.c

```diff
@@ -357,7 +357,7 @@ const mcuTypeInfo_t *getMcuTypeInfo(void)
 #elif defined(STM32H723xx) || defined(STM32H725xx)
         { .id = MCU_TYPE_H723_725, .name = "STM32H723/H725" },
 #elif defined(STM32G474xx)
```

Comment:
@haslinghuis : Why is STM32G474xx defined for STM32G473?

Reply:
Trying to add new target

```bash
#elif defined(STM32G473xx)
        { .id = MCU_TYPE_G473, .name = "STM32G473" },
#elif defined(STM32G474xx)
        { .id = MCU_TYPE_G474, .name = "STM32G474" },
```

and working on it

![image](https://github.com/user-attachments/assets/e6ecf47f-0f7c-4d3b-b242-cfd625364fbb)

got stuck at `ADC_Typedef` at the moment.

Reply:
Perhaps just use 

```bash
#elif defined(STM32G473xx) ||defined(STM32G474xx)
            { .id = MCU_TYPE_G47X, .name = "STM32G47X" },
```

to retain backwards compatibility

Reply:
Think 
```bash
#elif defined(STM32H723xx) || defined(STM32H725xx)
        { .id = MCU_TYPE_H723_725, .name = "STM32H723/H725" },
```

should be 

```bash
#elif defined(STM32H723xx) || defined(STM32H725xx)
        { .id = MCU_TYPE_H72X .name = "STM32H72X" },
```

Reply:
We can split those now... no need for X. Use the full identifier.